nomenclature/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 from nomenclature.codes import CodeList  # noqa
 from nomenclature.core import DataStructureDefinition, create_yaml_from_xlsx  # noqa
-from nomenclature.testing import assert_valid_yaml  # noqa
+from nomenclature.cli import cli  # noqa
```

Comment:
Maybe we want to additionally expose `assert_valid_yaml` and `assert_valid_structure` to the user.
`from nomenclature.testing import assert_valid_yaml, assert_valid_structure`

Reply:
I actually wanted to align the behavior more closely with pandas and numpy (i.e., the most sophisticated packages I know). There, the testing-methods like `assert_frame_equal` are not exposed at the top level, but can be imported from the module, see for example 

https://github.com/IAMconsortium/pyam/blob/732752c9c1af9ab453b28584a4a4e05cc9930c0a/tests/test_core.py#L110

Equivalently, with the new structure, a user can still do
```python
from nomenclature.testing import assert_valid_structure
```

Reply:
Ok, following numpy and pandas sounds like a very sensible choice.